docker-compose.yml

```diff
@@ -2,7 +2,9 @@ version: '2'
 services:
   base:
      volumes:
-       - ./:/var/lib/eventkit/
```

Comment:
Why is this being reverted? Was previously change so other files could be modified without rebuilding